src/types.ts

```diff
@@ -92,3 +92,25 @@ export function schemaTitle(schema: Schema): string {
     atf4: "ATF4 Scheme",
   }[schema];
 }
+
+// The singular form of the object (with the article) managed by this schema
+export function schemaSingularNoun(schema: Schema): string {
```

Comment:
So nice! I wonder if we could read this in from a .json config file to separate configuration from code as we did for the schemea.json files?

Reply:
We could put these and the title in the same .json defining the schema, yes. Maybe it does fit nicely there. OK if we think about that and defer a refactor? Still unsure how much we want to invest in this approach generally, especially now that we're running against govuk style best practices with the expanding form so much.

Reply:
> OK if we think about that and defer a refactor?

:+1: 

Yes this is a nice to have from my perspective.